### DISCUSSION TITLE
Option to install apps directly in Alpine LXC

### DISCUSSION THREAD

**Discussion by iodize6399** [tteck/Proxmox#2250] [General]
Can Adguard Home, currently offering installation options for Debian and Ubuntu LXC, also include Alpine OS?
I see some apps such as Docker, Vaultwarden, etc. being offered for Alpine LXC directly. Tailscale is yet another app (although not directly) that can be selected to be installed on an already created Alpine LXC.

**Comment by tteck** [comment]
The repository only accepts Alpine applications that make use of the Alpine Package Keeper.
https://pkgs.alpinelinux.org/packages?name=adguard&branch=edge&repo=&arch=x86_64&maintainer=
There is no package for Adguard Home.
